### PR TITLE
Feature/65 Added login cypress test for basic checks

### PR DESCRIPTION
**Pull request by swatiDgithub**

### CODE REVIEW

File: e2e/cypress/integration/login/login.spec.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    
+    it('Link to Apply to Drive',()=>{
+        cy.contains('Apply to Drive!')
+        .should('have.attr','href','/signup')
+    })
+    
+    it('requires username',()=>{
+        cy.findByLabelText(inputLabels.username).clear()
+        cy.findByLabelText(inputLabels.password).type("password")
+        cy.get('span.MuiButton-label').click() // clicking on Sign In button
```
Comment (l2silver) [inline e2e/cypress/integration/login/login.spec.js:34]: Can you use cy.findByText for the button.

**Review by l2silver — CHANGES_REQUESTED** [review]
Hey Swati! This is excellent. I've just got a few requirements. Can you include in your tests clicking the signin button for the success and failure states. And then comment them out when you're satisfied that they'll pass when the develop activates them again with the proper code.

File: e2e/cypress/integration/login/login.spec.js (hunk 1 above)
Comment (swatiDgithub) [reply e2e/cypress/integration/login/login.spec.js]: .findbyText returns multiple Sign In elements, I used findAllByText('Sign In').last().click()
Is this okay ?

Comment (l2silver) [reply e2e/cypress/integration/login/login.spec.js]: Yeah that's great.

File: e2e/cypress/integration/login/login.spec.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        cy.contains('Unable to log in')
+        
+    })
+    
+     Test to validate Valid login 
+    it('naviagates to next page after sucessfull login',()=>{
+        cy.findByLabelText(inputLabels.username).type(this.data.validUsername)
+        cy.findByLabelText(inputLabels.password).type(this.data.validPassword)
+        cy.findAllByText('Sign In').last().click() 
+        cy.url().should('include','/drivers-registration')
```
Comment (l2silver) [inline e2e/cypress/integration/login/login.spec.js:61]: last thing, can we change this from '/drivers-registration' to '/drivers-search'

**Review by l2silver — CHANGES_REQUESTED** [review]
Just one last change, and then I think we can merge in. Thanks.

**Review by l2silver — APPROVED** [review]

**Pull request closed without merging**